registration/templates/registration/password_reset_complete.html

```diff
@@ -6,7 +6,7 @@
 {% block content %}
 <p>
     {% trans "Your password has been reset!" %}
-    {% trans "You may now" %}<a href="{{ login_url }}">{% trans "log in" %}</a>.
+    {% trans "You may now" %} <a href="{{ login_url }}">{% trans "log in" %}</a>.
```

Comment:
I was trying to not change the existing gettext translation, to my opinion would be best for tiny change like thisone.

If the maintainer wants to change then maybe we could take this approach:
{% blocktrans with l_url=login_url %}You may now {{ l_url }} log in{% endblocktrans %}.